src/custom-element.js

```diff
@@ -497,6 +497,14 @@ function createBaseCustomElementClass(win) {
       ));
     }
 
+    /**
+     * Unloads the element.
+     * @final
+     */
+    unload() {
```

Comment:
Is this necessary somewhere? I don' think we should be exposing it.

Reply:
Moved it out of the CustomElement.